app/models/hearing_summary.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+class HearingSummary
+  include ActiveModel::Model
+
+  attr_accessor :body, :hearing_date
```

Comment:
I think we are missing where we pass `hearing_date` to the model as well

Reply:
Similar to the `defendant` model, the `hearing_date` isn't data we receive in the `body` of a hearing summary, it's supplemental data from a subsequent hearing call, so I've added it as an additional attribute that can be populated when that data is available.

In both cases maybe I shouldn't have included them yet, as they're part of the adaptations which are still to be done. If it's cleaner I can remove them from both models for now and add them in a later PR.

Reply:
That makes sense. I'm happy to have them in as it complements your serializer specs.